app/src/main/javascript/bundles/cell-type-wheel-heatmap/index.js

```diff
@@ -5,7 +5,8 @@ import CellTypeWheelMultiExperimentHeatmap from '@ebi-gene-expression-group/scxa
 
 const render = (options, target) => {
   ReactDOM.render(
-    <CellTypeWheelMultiExperimentHeatmap {...options} />,
+    <CellTypeWheelMultiExperimentHeatmap {...options}
+    />,
```

Comment:
We don't need a separate line here I think. Just format issue though.

Reply:
Fixed it